### PR TITLE
Added Some Status code

### PR DESCRIPTION
**Pull request by kiwioverflow**

### CODE REVIEW

File: src/api/auth-api-client.ts [hunk 1]
```diff
@@ -98,19 +98,47 @@ export interface TokenLoginForm extends LoginForm {
...
+  TOO_MANY_REQUEST_AT_A_TIME = -30,
+  MISMATCH_PASSCODE = -31,
+  EXCEED_DAILY_REQUEST_LIMIT = -32,
+  EXCEED_DAILY_REQUEST_LIMIT_VOICECALL = -33,
+  EXCEED_DAILY_REQUEST_LIMIT_WITHOUT_TOKEN = -34,
   DEVICE_NOT_REGISTERED = -100,
   ANOTHER_LOGON = -101,
   DEVICE_REGISTER_FAILED = -102,
   INVALID_DEVICE_REGISTER = -110,
-  INCORRECT_PASSCODE = -111,
```
Comment (JellyBrick) [inline src/api/auth-api-client.ts:110]: BC Break?

**Review by kiwioverflow — COMMENTED** [review]
👍

File: src/api/auth-api-client.ts (hunk 1 above)
Comment (kiwioverflow) [reply src/api/auth-api-client.ts:110]: > BC Break?
> @JellyBrick 

Not a major change, but need to unify the words with other status codes.

Comment (kiwioverflow) [reply src/api/auth-api-client.ts:110]: @JellyBrick 
or how about make two for legacy support?

`INCORRECT_PASSCODE = -111,
INVALID_PASSCODE = -111,`

**Review by JellyBrick — APPROVED** [review]

**Pull request merged**